drivers/docker/docklog/docker_logger_test.go

```diff
@@ -13,22 +14,34 @@ import (
 	"golang.org/x/net/context"
 )
 
+func testContainerDetails() (string, string, string) {
+	if runtime.GOOS == "windows" {
+		return "dantoml/busybox-windows:08012019",
```

Comment:
is there not a more official busybox-windows?

Reply:
I found one, but it was built on a base container that didn't match our (kernel?) version